### DISCUSSION TITLE
Where can I find the logs of the application

### DISCUSSION THREAD

**Discussion by jundayin** [darrenburns/posting#78] [Q&A] [answered]
I encounter some error when sending form request but error message is to obscure to me to figure out whether if I'm missing something or if it is bug in the application. It says something like `ValueError: Invalid placeholder in string` and no stacktrace provided

**Comment by darrenburns** [comment, marked as answer]
There are no logs yet. It's one of the next things I'm going to add - so hopefully within the next month or two.
